cupy/cuda/cub.pyx

```diff
@@ -4,7 +4,9 @@
 
 import numpy
 
-from cupy.core.core cimport ndarray, _internal_ascontiguousarray
+from cupy.core.core cimport (ndarray, _internal_ascontiguousarray,  # noqa:E211
+                             _internal_asfortranarray)
```

Comment:
I'd rather use two `from` statements than a `noqa`

Reply:
Fixed in 9cc059f.